src/modules/service-catalog/components/service-catalog-list-item/Search.tsx

```diff
@@ -26,12 +42,34 @@ export const Search = ({
 }: SearchProps) => {
   const { t } = useTranslation();
 
+  const SearchEndIcon = () => {
+    if (!isLoading && searchInputValue) {
+      return (
+        <Tooltip
+          content={t("service-catalog.clear-search", "Clear search")}
+          placement="bottom"
+          size="small"
+        >
+          <IconButton
```

Comment:
I think we need an aria-label for a11y, even if we already have the tooltip. In the Admin part we omitted aria labels when there is already a tooltip, but then we received some reports from a11y that we need an aria-label as well.